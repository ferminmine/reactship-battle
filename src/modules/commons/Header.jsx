import React from 'react';
import withStyles from 'react-jss';

const styles = {
  headerContainer: {
    display: 'flex',
    justifyContent: 'center'
  },
  headerImage: {
    height: '13em',
    width: '60%'
  },
  '@media (max-width: 650px)': {
    headerImage: {
      width: '100%'
    }
  }
};

class Header extends React.Component {
  render = () => {
    const { classes } = this.props;
    return (
      <div className={classes.headerContainer}>
        <img src="/imgs/reactship.png" alt="App Header" className={classes.headerImage} />
      </div>
    );
  };
}

export default withStyles(styles)(Header);
