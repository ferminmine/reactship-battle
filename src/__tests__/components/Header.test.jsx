import React from 'react';
import { mount } from 'enzyme';
import Header from '../../modules/commons/Header';
import renderer from 'react-test-renderer';

describe('Header Component Test', () => {
  it('renders without crashing', () => {
    mount(<Header />);
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
