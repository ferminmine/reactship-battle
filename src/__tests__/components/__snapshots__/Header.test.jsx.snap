// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Test matches the snapshot 1`] = `
<div
  className="Header-headerContainer-0-1-1"
>
  <img
    alt="App Header"
    className="Header-headerImage-0-1-2"
    src="/imgs/reactship.png"
  />
</div>
`;
